### PR TITLE
Adds auto repeat feature

### PR DESCRIPTION
**Pull request by DimaNaumov**

### CODE REVIEW

File: test/visualTest.ts [hunk 1]
```diff
@@ -144,6 +144,26 @@ namespace powerbi.extensibility.visual.test {
...
+                it("repeat animation", (done) => {
+                    dataView.metadata.objects = {
+                        playback: {
+                            repeat: true
+                        }
+                    };
+                    visualBuilder.updateRenderTimeout(dataView, () => {
+                        visualBuilder.animationPlay.d3Click(5, 5);
+                        let animatedDot = visualBuilder.dotsContainerDot;
+                        debugger;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: It'd be better to remove this statement.

File: test/visualTest.ts [hunk 2]
```diff
@@ -144,6 +144,26 @@ namespace powerbi.extensibility.visual.test {
...
 
+                it("repeat animation", (done) => {
+                    dataView.metadata.objects = {
+                        playback: {
+                            repeat: true
+                        }
+                    };
+                    visualBuilder.updateRenderTimeout(dataView, () => {
+                        visualBuilder.animationPlay.d3Click(5, 5);
+                        let animatedDot = visualBuilder.dotsContainerDot;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please specify a type

File: test/visualTest.ts [hunk 3]
```diff
@@ -144,6 +144,26 @@ namespace powerbi.extensibility.visual.test {
...
+                        debugger;
+                        visualBuilder.updateRenderTimeout(dataView, () => {
+                            visualBuilder.animationPlay.d3Click(5, 5);
+                            let currentAnimatedDot = visualBuilder.dotsContainerDot;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please specify a type

File: test/visualTest.ts (hunk 1 above)
Comment (DimaNaumov) [inline test/visualTest.ts]: Ooops

**Pull request merged**